src/components/LandingPage.tsx

```diff
@@ -3,54 +3,114 @@ import { Button, Container, Grid, Icon, Label, Link, List } from '../components'
 import { Image } from 'semantic-ui-react';
 import timage from '../images/landing-background-top.png';
 import bimage from '../images/landing-background-bot.png';
+import { RootState } from 'store';
+import { useSelector } from 'react-redux';
+import * as utils from '../utils';
+
+const LandingPage = ({}: {}): any => {
+  const currentUser = useSelector((state: RootState) => state.auth.user);
+  const currentUserScope = useSelector((state: RootState) => state.auth.scope);
+
+  const isVolunteer =
+    currentUser &&
+    utils.hasPermission({ currentScope: currentUserScope, requiredScope: ['Volunteer'] });
+  const user = currentUser?.firstName;
+  const rolename = currentUser?.roleName;
 
-const LandingPage = ({ user }: { user: string }): any => {
   return (
     <React.Fragment>
-      <div className="videodesc">
-        <Image src={timage} fluid rounded />
+      {isVolunteer ? (
+        <div className="videodesc">
+          <Image src={timage} fluid rounded />
+
+          <h2>
+            <Icon size="big" name="user" color="teal" float="left" />
+            {'Hello ' + user}
+          </h2>
+          <p>
+            <br />
+            Welcome to the MBT timestamping tool.{' '}
+          </p>
+          <p>
+            Thank you for your interest and contribution to the timestamping project. You can start
+            by selecting any video from the list on the right. Or you can search for videos using
+            video titles, video ID or any keywords.
+          </p>
+          {user === 'Guest' && <p>Please log in before you begin.</p>}
+          <p>
+            You can timestamp already processed videos. List them by unchecking the{' '}
+            <strong>Hide Processed Videos.</strong>
+          </p>
+          <p>
+            See here for a brief tutorial on timestamping : &nbsp;
+            <a
+              href="https://drive.google.com/file/d/1iEbK2paCpbjqGIEusM5YBQL-SaA9Z2lM/view"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              MBT Timestamping Guide
+            </a>
+          </p>
+          <p>
+            You can ask for help or offer your comments, feedback and suggestions on our Slack
+            channel here : &nbsp;
+            <a
+              href="https://app.slack.com/client/TK00UBC3W/C014QD9C4SE?cdn_fallback=1"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              MBT Timestamping Volunteers Channel on Slack
+            </a>
+          </p>
+          <p>
+            <br />
+            <br />
+            <br />
+            <small>
+              User Info: <br />
+              {'Scope : ' + (currentUserScope && currentUserScope[1])} <br />
```

Comment:
I'm not sure that the word `Scope` will have much meaning to users and could be confusing. I'm guessing that you are intending to display a group name that the user is in, however I don't think there is a guarantee that the index you are referring to will contain that name. Could you clarify what information you would like to display to the user here?

Reply:
I was trying to display their group.

Reply:
Okay, for now maybe we can remove this and later we can display a list of groups the user belongs to in their profile page, or if you think it would be really helpful we can make another PR later that fetches the user's groups and displays it.

Reply:
Removed. Edited the file here, the deploy preview looks ok.